3_javascript/javascript2/week1/script.js

```diff
@@ -0,0 +1,77 @@
+// Find and count the Danish letters
+function danishLetters(vowels) {
+  let obj = {};
```

Comment:
the function could be made more efficient for example by using an object to keep track of the numbers:   ` const danishVowels = { "æ": 0, "ø": 0, "å": 0 }`.  Also would be nice to check for capital letters as well.